src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -2,10 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
+using System.Security;
 using System.Windows;
+using Microsoft.VisualStudio.Experimentation;
```

Comment:
fyi, I think these aren't sorted, and will fail the roslyn formatters

Reply:
although your build passed so maybe it's not a rule violation?

Reply:
They are sorted based on the "System using directives go first" then and in alphabetical order. I sampled a few other files in our repo and they are sorted that way as well. What am I missing?

Reply:
I checked again and they are already sorted based on the rules mentioned above. Closing comment but feel free to reopen if you have other concerns.